ibmnode/8/sdk/linux/x86_64/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:16.04
+
+MAINTAINER Bethany Griggs <bethany.griggs@uk.ibm.com> (@BethGriggs)
+
+RUN buildDeps='wget ca-certificates unzip openjdk-8-jre'; \
+    apt-get update \
+    && apt-get install -y --no-install-recommends \
+    $buildDeps \
+    && rm -rf /var/lib/apt/lists/* \
+    set -eux; \
+    IBM_VERSION="8.1.4.0" \
+    NODE_VERSION="8.1.4" \
+    ARCH="$(dpkg --print-architecture)"; \
+    case "${ARCH}" in \
+       amd64|x86_64) \
```

Comment:
If you expect to have multi-arch dockerfiles eventually, suggest you have the directory structure as follows

ibmnode/8sdk/linux/ubuntu/Dockerfile
ibmnode/8/sdk/linux/alpine/Dockerfile

The same Dockerfile can then support all arches and you need to add additional arches to your case statement here.

Reply:
By multi-arch do you mean multi-os? These dockerfiles are already multi-arch. Assuming you mean multi-os (having alpine images as well as ubuntu images). Seems easier to just have a super-simple directory structure for now, then it's nice and easy to find things. If we add alpine later we can just do:

```
ibmnode/ubuntu/Dockerfile-8
ibmnode/alpine/Dockerfile-8
```

What's the need for having a four level deep directory structure? Seems like it'll just make everything harder to find.